examples/computer_vision/unets_tf_keras/startup-hook.sh

```diff
@@ -1,4 +1,3 @@
 pip install git+https://github.com/tensorflow/examples.git
 pip install -q -U tfds-nightly
-python -m tensorflow_datasets.scripts.download_and_prepare --register_checksums=True --datasets='oxford_iiit_pet:3.*.*' --data_dir='/tmp/data'
```

Comment:
Doing this in a startup hook was dtrain-safe, and removing this seems like it could cause problems.

Reply:
Why is doing this in a startup-hook dtrain-safe?

Reply:
If two processes are trying to download to the same data location at the same time, who ensures that they are all written correctly?